Failed to build on FreeBSD

```
$ cmake -S . -B build
-- The C compiler identification is Clang 18.1.5
-- The CXX compiler identification is Clang 18.1.5
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Configuring done (2.7s)
-- Generating done (0.0s)
-- Build files have been written to: /work/covscript/build
```

```
$ gmake -C build
gmake: Entering directory '/work/covscript/build'
[  3%] Building CXX object CMakeFiles/covscript.dir/sources/compiler/codegen.cpp.o
[  6%] Building CXX object CMakeFiles/covscript.dir/sources/compiler/compiler.cpp.o
[  9%] Building CXX object CMakeFiles/covscript.dir/sources/compiler/lexer.cpp.o
[ 12%] Building CXX object CMakeFiles/covscript.dir/sources/compiler/parser.cpp.o
[ 15%] Building CXX object CMakeFiles/covscript.dir/sources/instance/type_ext.cpp.o
[ 18%] Building CXX object CMakeFiles/covscript.dir/sources/instance/instance.cpp.o
[ 21%] Building CXX object CMakeFiles/covscript.dir/sources/instance/runtime.cpp.o
[ 25%] Building CXX object CMakeFiles/covscript.dir/sources/instance/statement.cpp.o
[ 28%] Building CXX object CMakeFiles/covscript.dir/sources/system/common.cpp.o
/work/covscript/sources/system/common.cpp:44:2: warning: Compatible Mode for Unix Platform [-W#warnings]
   44 | #warning Compatible Mode for Unix Platform
      |  ^
In file included from /work/covscript/sources/system/common.cpp:47:
/work/covscript/sources/system/./unix/common.cpp:188:2: warning: Compatible Mode for Linux System [-W#warnings]
  188 | #warning Compatible Mode for Linux System
      |  ^
/work/covscript/sources/system/./unix/common.cpp:201:2: warning: Compatible Mode for Linux System [-W#warnings]
  201 | #warning Compatible Mode for Linux System
      |  ^
3 warnings generated.
[ 31%] Building CXX object CMakeFiles/covscript.dir/sources/covscript.cpp.o
/work/covscript/sources/covscript.cpp:304:49: error: use of undeclared identifier 'COVSCRIPT_PLATFORM_HOME'
  304 |                 base_path += cs::path_delimiter + std::string(COVSCRIPT_PLATFORM_HOME) + cs::path_separator + "imports";
      |                                                               ^
1 error generated.
gmake[2]: *** [CMakeFiles/covscript.dir/build.make:202: CMakeFiles/covscript.dir/sources/covscript.cpp.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:95: CMakeFiles/covscript.dir/all] Error 2
gmake: *** [Makefile:91: all] Error 2
gmake: Leaving directory '/work/covscript/build'
```

--- a/include/covscript/core/definition.hpp
+++ b/include/covscript/core/definition.hpp
@@ -52,6 +52,12 @@
 #define COVSCRIPT_PLATFORM_HOME "/Applications/CovScript.app/Contents/MacOS/covscript"
 #endif
 
+#if defined(__FreeBSD__)
+#define COVSCRIPT_PLATFORM_UNIX
+#define COVSCRIPT_PLATFORM_NAME "FreeBSD"
+#define COVSCRIPT_PLATFORM_HOME "/usr/local/share/covscript"
+#endif
+
 #if defined(COVSCRIPT_PLATFORM_LINUX) || defined(COVSCRIPT_PLATFORM_DARWIN)
 #define COVSCRIPT_PLATFORM_UNIX
 #endif

--- a/include/covscript/core/version.hpp
+++ b/include/covscript/core/version.hpp
@@ -37,8 +37,8 @@
 *   the last two digits are different, it will be regarded as a compatible
 *   version.
 */
-#define COVSCRIPT_VERSION_NUM 3,4,3,12
-#define COVSCRIPT_VERSION_STR "3.4.3 Castor fiber(Stable) Build 12"
+#define COVSCRIPT_VERSION_NUM 3,4,3,13
+#define COVSCRIPT_VERSION_STR "3.4.3 Castor fiber(Stable) Build 13"
 #define COVSCRIPT_STD_VERSION 210508
 #define COVSCRIPT_API_VERSION 230801
 #define COVSCRIPT_ABI_VERSION 230308

